src/language_server.ts

```diff
@@ -394,6 +394,9 @@ export class LanguageServer {
 				}
 				this.client = undefined;
 			}
+			if(this.lsOutputChannel){
+				this.lsOutputChannel.dispose();
```

Comment:
Does this mean we immediately clear the logs when we stop the server? Is this always desirable? Would it be feasible to just append to the old logs when we restart the server? Possibly with a clear separator.

Reply:
yes that's what it currently does. I though that was what we wanted. I'll try to see if I can make it append as I like it better to.

Reply:
I'm honestly not sure, I don't think we discussed the desired behavior in depth so I think we have some freedom here.

Reply:
![video](https://user-images.githubusercontent.com/95339363/224957125-a4fe8e6a-01af-48f7-a3e2-02517e48c731.gif)

Do you think this is enough of a separator?

Reply:
I think so: the line draws the eye, and then the "Started ..." and "Stopped ..." messages clearly show what it means. I like it.